src/main/java/uk/gov/register/presentation/representations/NewRepresentationWriter.java

```diff
@@ -27,15 +30,21 @@ public final long getSize(View t, Class<?> type, Type genericType, Annotation[]
 
     @Override
     public boolean isWriteable(Class<?> type, Type genericType, Annotation[] annotations, MediaType mediaType) {
-        return NewEntryView.class.isAssignableFrom(type);
+        return NewEntryView.class.isAssignableFrom(type) ||
+                ItemView.class.isAssignableFrom(type);
     }
 
     @Override
     public void writeTo(View view, Class<?> type, Type genericType, Annotation[] annotations, MediaType mediaType, MultivaluedMap<String, Object> httpHeaders, OutputStream entityStream) throws IOException, WebApplicationException {
         if (view instanceof NewEntryView) {
             writeEntryTo(entityStream, requestContext.getRegister().getFields(), ((NewEntryView) view).getEntry());
         }
+        if (view instanceof ItemView) {
+            writeItemTo(entityStream, ((ItemView) view).getContent());
+        }
```

Comment:
@om-sharma raised concerns IRL that this `if (foo instanceof Bar)` chain is going to result in ugly code as we add more resources to the product.  He suggested instead that, given we already have a way of producing JSON for each resource, in principle all we should need to have is a generic way of producing another representation from JSON and then we don't need this special casing.

I'm not sure I agree but I'll investigate and I wanted to capture this context on the PR. I think we both agreed that this approach might not work for the turtle type.